src/static/countryNames/es.json

```
@@ -0,0 +1,256 @@
+    "MQ": "Martinica",
+    "MR": "Mauritania",
+    "MU": "Mauricio",
+    "YT": "Mayotte",
+    "MX": "México",
+    "FM": "Micronesia, Estados Federados de",
+    "MD": "Moldavia",
+    "MC": "Mónaco",
+    "MN": "Mongolia",
+    "MS": "Montserrat",
+    "MA": "Marruecos",
+    "MZ": "Mozambique",
+    "MM": "Myanmar",
+    "NA": "Namibia",
+    "NR": "Nauru",
+    "NP": "Nepal",
+    "NL": "Países Bajos",
+    "NC": "Nueva Caledonia",
+    "NZ": "Nueva Zelanda",
+    "NI": "Nicaragua",
+    "NE": "Níger",
+    "NG": "Nigeria",
+    "NU": "Niue",
+    "NF": "Isla Norfolk",
+    "MK": "Macedonia del Norte",
+    "MP": "Islas Marianas del Norte",
+    "NO": "Noruega",
+    "OM": "Omán",
+    "PK": "Pakistán",
+    "PW": "Palau",
+    "PS": "Palestina",
+    "PA": "Panamá",
+    "PG": "Papúa Nueva Guinea",
+    "PY": "Paraguay",
+    "PE": "Perú",
+    "PH": "Filipinas",
+    "PN": "Pitcairn",
+    "PL": "Polonia",
+    "PT": "Portugal",
+    "PR": "Puerto Rico",
+    "QA": "Catar",
+    "RE": "Reunión",
+    "RO": "Rumanía",
+    "RU": "Rusia",
+    "RW": "Ruanda",
+    "SH": "Santa Helena",
+    "KN": "San Cristóbal y Nieves",
+    "LC": "Santa Lucía",
+    "PM": "San Pedro y Miquelón",
+    "VC": "San Vicente y las Granadinas",
+    "WS": "Samoa",
+    "SM": "San Marino",
+    "ST": "Santo Tomé y Príncipe",
+    "SA": "Arabia Saudita",
+    "SN": "Senegal",
+    "SC": "Seychelles",
+    "SL": "Sierra Leona",
+    "SG": "Singapur",
+    "SK": "Eslovaquia",
+    "SI": "Eslovenia",
+    "SB": "Islas Salomón",
+    "SO": "Somalia",
+    "ZA": "Sudáfrica",
+    "GS": "Islas Georgias del Sur y Sandwich del Sur",
+    "ES": "España",
+    "LK": "Sri Lanka",
+    "SD": "Sudán",
+    "SR": "Surinam",
+    "SJ": "Svalbard y Jan Mayen",
+    "SZ": "Esuatini",
+    "SE": "Suecia",
+    "CH": "Suiza",
+    "SY": "República Árabe Siria",
+    "TW": "Taiwán",
+    "TJ": "Tayikistán",
+    "TZ": "Tanzania",
+    "TH": "Tailandia",
+    "TL": "Timor-Leste",
+    "TG": "Togo",
+    "TK": "Tokelau",
+    "TO": "Tonga",
+    "TT": "Trinidad y Tobago",
+    "TN": "Túnez",
+    "TR": "Turquía",
+    "TM": "Turkmenistán",
+    "TC": "Islas Turcas y Caicos",
+    "TV": "Tuvalu",
+    "UG": "Uganda",
+    "UA": "Ucrania",
+    "AE": "Emiratos Árabes Unidos",
+    "GB": "Reino Unido",
+    "US": "Estados Unidos",
+    "UM": "Islas Ultramarinas Menores de los Estados Unidos",
+    "UY": "Uruguay",
+    "UZ": "Uzbekistán",
+    "VU": "Vanuatu",
+    "VE": "Venezuela",
+    "VN": "Vietnam",
+    "VG": "Islas Vírgenes Británicas",
+    "VI": "Islas Vírgenes de los Estados Unidos",
+    "WF": "Wallis y Futuna",
+    "EH": "Sahara Occidental",
+    "YE": "Yemen",
+    "ZM": "Zambia",
+    "ZW": "Zimbabue",
+    "AX": "Islas Åland",
+    "BQ": "Bonaire, Sint Eustatius y Saba",
+    "CW": "Curaçao",
+    "GG": "Guernsey",
+    "IM": "Isla de Man",
+    "JE": "Jersey",
+    "ME": "Montenegro",
+    "BL": "San Bartolomé",
+    "MF": "San Martín (parte francesa)",
```

Wdyt if we change this one to 
```suggestion
    "MF": "San Martín (Francia)",
```